Dev-Corner/slides/toolshed.html

```diff
@@ -0,0 +1,861 @@
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### <i class="fa fa-bullseye" aria-hidden="true"></i> Objectives
+
+                            - Discover what is a Tool Shed
+                            - Be able to install tools and workflows from a Tool Shed into a Galaxy instance
+                            - Be able to publish tools with Planemo
+                       </script>
+                    </section>
+                </section>
+                <section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ## 1. What is a Tool Shed?
+                            <img src="../images/toolShed.jpg" class="no-border" width="40%">
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### <img src="../images/toolShed.jpg" class="no-border"> = Galaxy App Store
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### <img src="../images/toolShed.jpg" class="no-border"> = Galaxy App Store
+                            - Galaxy tools
+                            - ~~Recipes for installing and compilling tools~~,
+                            - Data Managers,
+                            - Custom datatypes,
+                            - Exported workflows.
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### <img src="../images/toolShed.jpg" class="no-border"> = Galaxy App Store
+
+                            - For Galaxy administrators, facilitates:
+                              - **installing/updating** tools
+
+                            - For tool developers, facilitates:
+                              - **sharing** of Galaxy utilities
+                              - **versioning**
+
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Available Tool Sheds
+                            - [Galaxy community Tool Shed](https://toolshed.g2.bx.psu.edu/) -> Main Tool Shed
+                            - [Galaxy community test Tool Shed](https://testtoolshed.g2.bx.psu.edu/) -> Sandbox for testing
+                            - [List of public Tool Sheds](https://wiki.galaxyproject.org/PublicGalaxyServers#Public_ToolSheds)
+                            - [IFB Tool Shed](http://toolshed.france-bioinformatique.fr/)
```

Comment:
I think we can remove the IFB TS.
Add it in the Public TS if needed and broadcast the URL to the French community if neeeded.
... if needed 😖